.gitignore

```diff
@@ -30,3 +30,7 @@
 *.exe
 *.out
 *.app
+
+# Local folders
+/bin
+/build
```

Comment:
Wait... why do we have such a monster of a `.gitignore` file to begin with? :confused: 

I can get on board with adding `build/`. While I don't do it, I know that strange people like @stephenswat (:stuck_out_tongue:) like to put that into the source directory. But what about `bin/`? And what about all those other silliness-es that were already in the file? :confused:

Reply:
Let me remove ```bin/```. I leave the rest for you to decide :)